File: first_package/src/listener.cpp
#include "ros/ros.h" 
#include "std_msgs/String.h" 
  
void first_topic_callback(const std_msgs::String::ConstPtr& msg) 
{ 
  ROS_INFO("Listener heard: [%s]",msg->data.c_str()); 
} 
int main(int argc, char **argv) 
{ 
   ros::init(argc, argv, "listener"); 
   ros::NodeHandle n; 
   ros::Subscriber sub = n.subscribe("first_topic", 10, first_topic_callback); 
   ros::spin(); 
  
   return 0; 
 }

File: first_package/scripts/listener.py
#!/usr/bin/env python
import rospy
from std_msgs.msg import String

#first_topic callback
def first_topic_cb(data):
    rospy.loginfo("Listener heard: [%s]", data.data)
  
if __name__ == '__main__':
    rospy.init_node('listener', anonymous=True)
    rospy.Subscriber("first_topic", String, first_topic_cb)
    rospy.spin()
